west.yml

```diff
@@ -280,7 +280,7 @@ manifest:
       revision: 2b498e6f36d6b82ae1da12c8b7742e318624ecf5
       path: modules/lib/gui/lvgl
     - name: mbedtls
-      revision: 2f24831ee13d399ce019c4632b0bcd440a713f7c
+      revision: pull/60/head
```

Comment:
This can be removed as https://github.com/zephyrproject-rtos/mbedtls/pull/60 is now in `main`.